plugin/PDFDocumentPage.qml

```diff
@@ -40,19 +39,60 @@ DocumentPage {
         document: pdfDocument;
 
         onClicked: base.open = !base.open;
+
+        ViewPlaceholder {
+            enabled: pdfDocument.failure || pdfDocument.locked;
+            y: (flickable ? flickable.originY : 0)
+                   + (base.height - height - (passwd.visible ? passwd.height : 0)) / 2
+            text: if (pdfDocument.failure) {
+                //% "Broken file"
+                return qsTrId("sailfish-office-me-broken-pdf");
+            } else {
+                //% "Password protected file"
+                return qsTrId("sailfish-office-me-locked-pdf");
+            }
```

Comment:
Ok, I've removed the trailing ";" in QML lines I've wrote myself. They are still on the original lines of the file to keep a commit just related to the matter. I've also used the inlined "if" for the strings, as you suggested.